src/ui/overview.rs

```diff
@@ -81,14 +85,14 @@ fn draw_cli_version_block<B: Backend>(f: &mut Frame<B>, app: &mut App, area: Rec
 
     let table = Table::new(rows)
       .block(block)
-      .widths(&[Constraint::Percentage(50), Constraint::Percentage(50)]);
+      .widths(&[Constraint::Length(15), Constraint::Length(15)]);
```

Comment:
Not sure about using length as I had issues with it and hence used percentage, any reason you wanted to change this?

Reply:
Oh, yes, in this case 50/50 is also fine!